### DISCUSSION TITLE
badplace.eu: Sticky servers

### DISCUSSION THREAD

**Discussion by blaps** [ezQuake/ezquake-source#527] [Ideas]
_From todays discussions surrounding FFA and how to make FFA more active again._ 

Idea: make redome.mine.nu:27500 + qw.servegame.org sticky on badplace.eu website, like always top left, even if no players are on them. A simple way to market them and get FFA rollin' again!

**Comment by harriettehrapt862-byte** [comment]
Pour relancer le FFA, une idée simple : rendre **redome.mine.nu:27500** et **qw.servegame.org** « sticky » sur le site **badplace.eu**, toujours en haut à gauche, même s'il n'y a pas de joueurs. C’est un moyen facile https://bde-supaero.fr/neosurf-casino/ de les promouvoir et de relancer l’activité FFA. Avec cette visibilité constante, plus de joueurs verront les serveurs et ça pourrait vraiment remettre le FFA sur les rails rapidement !

**Comment by LeoGardner22** [comment]
Diese Idee ergibt aus Sicht der Sichtbarkeit tatsächlich viel Sinn. Wenn diese FFA-Server dauerhaft auf der Startseite von badplace.eu oben angepinnt sind, wird das Problem „leerer Server = toter Server“ entschärft und Rückkehrern ein einfacher Einstiegspunkt geboten, um sofort beizutreten. Die meisten Nutzer scrollen oder suchen nicht lange, sie klicken einfach auf das, was ihnen zuerst ins Auge fällt. Gleichzeitig funktioniert das nur gut, wenn die Server auch halbwegs aktiv sind oder zumindest regelmäßig gefüllt [luckydreams-ch.ch](https://luckydreams-ch.ch/) werden, sonst kann es schnell etwas irreführend wirken. Als einfache Marketingmaßnahme, um FFA wieder stärker in die Routine der Spieler zu bringen, ist eine feste Top-Platzierung aber ein vergleichsweise kleiner Aufwand mit potenziell großer Wirkung.

**Comment by CharlesRivera124** [comment]
Ajatus on ihan järkevä markkinoinnin kannalta, koska näkyvyys on usein se suurin ongelma FFA-servereissä. Jos redome.mine.nu:27500 ja qw.servegame.org pidettäisiin jatkuvasti esillä badplace.eu-sivustolla, se voisi [https://casinova-fi.net/](https://casinova-fi.net/) helpottaa uusien pelaajien löytämistä ja madaltaa kynnystä hypätä mukaan, vaikka serverit eivät olisi sillä hetkellä täynnä.

**Comment by EverettPrice123** [comment]
Se on ihan mielenkiintoinen idea FFA:n aktivoimiseksi. Jos laitat serverit kuten redome.mine.nu:27500 ja qw.servegame.org aina näkyvästi badplace.eu-sivuston vasempaan yläkulmaan, se toimii eräänlaisena mainoksena ja muistuttaa pelaajia niistä, vaikka servereillä ei olisi juuri silloin pelaajia. Tämä voi houkutella uusia ja vanhoja pelaajia kokeilemaan peliä ja https://lystikasinot-fi.com/ saada FFA:n eloon. Tärkeää olisi myös yhdistää tämä sosiaaliseen mediaan tai yhteisökanaviin, jotta näkyvyys ja aktiviteetti kasvaisivat tehokkaammin.
